pkg/rsafunc/rsafunc.go

```diff
@@ -1,4 +1,4 @@
-package krypto
+package rsafunc
```

Comment:
I don't know if I like the name `rsafunc`, I didn't want to use `rsa` because it's such a common go package and would need to alias everywhere. What it really is is a collection of high level RSA operations with the parameters hardcoded. Maybe call it `rsaop`? Or maybe just rsa with an alias? Trying to avoid `rsautil` or `rsahelper` because convention says to avoid names like that 🤷

Reply:
I can't think of any better names than what you've come up with so far, so any will do for me. Calling it `rsa` and aliasing wouldn't bother me either.

Reply:
I think I'm good with anything here. I could bike shed over names, but that's such quintessential bike shedding I've deleted my draft comments twice already. 😆